.github/workflows/main.yml

```
@@ -42,7 +42,7 @@ jobs:
       matrix:
         python:
         - "3.9"
-        - "3.13"
+        - ">=3.13.5"
```

I suggest adding a note to inform potential conflict resolution with `skeleton`:

```suggestion
        # - "3.13"
        - ">=3.13.5"  # temporary bound until it becomes the default, python/cpython#135151
```